### PR TITLE
set FindBy locator for pickupLocationInput

### PR DESCRIPTION
**Pull request by pzozuliak**
us1 petro

### CODE REVIEW

File: code/frontend/web-app/src/test/java/edu/githubproject/app/web/tools/pages/MainPage.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
     @FindBy() // TODO Nurlan N
     public WebElement rideTab;
 
-    @FindBy() // TODO Petro Z
+    @FindBy(xpath = "//div[contains(@class,'tm tn to')]") // TODO Petro Z
```
Comment (JaneRomaniuk) [inline code/frontend/web-app/src/test/java/edu/githubproject/app/web/tools/pages/MainPage.java:23]: Good job!

**Review by VolodymyrSDET — APPROVED** [review]
GOOD JOB PETRO!!!!!!!!!!!!

**Review by JaneRomaniuk — APPROVED** [review]
👍 LGTM

**Review by calicodegirl — APPROVED** [review]
👍 LGTM!

**Comment by giabakubat** [comment]
👍LGTM!

**Pull request merged**